src/api/service/secret-manager.ts

```diff
@@ -0,0 +1,136 @@
+
+import * as fs from 'fs'
+import {homedir} from 'os'
+import * as util from 'util'
+import * as inquirer from 'inquirer'
+
+const prompt = inquirer.createPromptModule()
+
+class Secret {
+  private _value: string
+
+  constructor(value: string) {
+    this._value = value
+    Object.defineProperty(this, 'inspect', {
+      enumerable: false,
+      configurable: false,
+      writable: false,
+      value: function () {
+        return this.toString()
+      },
+    })
+  }
+
+  public getPlainText(): string {
+    return this._value
+  }
+
+  public toString = (): string => {
+    return '*****'
+  }
+
+  /**
+   * Hide values from util.inspect, console.log, console.dir.
+   * @returns {string} Masked value
+   */
+  // [util.inspect.custom](): string {
+  //   return this.toString()
+  // }
+}
+
+class EntryAccessor {
```

Comment:
Multiple object in a single file. Please seperate.

Reply:
We will need to revisit this class and find a good abstraction/architecture